app/src/main/java/edu/uwstout/p2pchat/FileTransfer/SendDataService.java

```diff
@@ -220,23 +217,15 @@ private void transmitINetAddress(@NonNull final Intent intent)
 
             Log.d(LOG_TAG, "Client socket connected: " + socket.isConnected());
             OutputStream outputStream = socket.getOutputStream();
-            InputStream inputStream = null;
-
-            try
+            InputStream inputStream = serializeINetAddress(LOCALHOST);
+            // this is where the data is sent
+            if (inputStream != null)
```

Comment:
Like I mentioned earlier, maybe throw instead? It would be really bad if inputStream were to be null here. And it shouldn't happen with proper testing.

Reply:
Like I mentioned earlier, without changing our error handling behavior, that just changes where the logging occurs.